layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,93 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/dipesh-rawat/downloadkubernetes/integrate-k8s-website/dist/release_binaries.json" }}
+
+{{ $Binaries := slice }}
+{{ $AllOSes := slice }}
+{{ $AllArch := slice }}
+{{ $AllVersions := slice }}
+
+{{ range $key, $value := $response }}
+    {{ if eq $key "Binaries" }}
+        {{ $Binaries = $value }}
+    {{ else if eq $key "AllOSes" }}
+        {{ $AllOSes = $value }}
+    {{ else if eq $key "AllArch" }}
+        {{ $AllArch = $value }}
+    {{ else if eq $key "AllVersions" }}
+        {{ $AllVersions = $value }}
+    {{ end }}
+{{ end }}
+
+<link href="{{ "css/release_binaries.css" | relURL }}" rel="stylesheet">
+<script defer src="{{ "js/release_binaries.js" | relURL }}"></script>
+
+<div class="filter-options">
```

Comment:
Maybe:
```suggestion
<div class="binaries-download-filter-option">
```

Consider wrapping all the filter options in a [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) element. If the details is collapsed, you have a choice of what to implement. Options include:

- hide **all** the binaries details when the filters `<div>` element is collapsed
- treat a collapsed filters `<div>` element as “show everything”
- only collapse / uncollapse the architecture chooser, and default it to `runtime.PlatformArch` if collapsed

Reply:
I've considered the points from this suggestion and add them into the latest commit.